kornia/augmentation/_2d/intensity/motion_blur.py

```diff
@@ -81,17 +81,23 @@ def __init__(
         self._param_generator = rg.MotionBlurGenerator(kernel_size, angle, direction)
         self.flags = dict(border_type=BorderType.get(border_type), resample=Resample.get(resample))
 
-    def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, Tensor]:
+    def generate_parameters(self, batch_shape: torch.Size):
```

Comment:
```suggestion
    def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, Tensor]:
```
I think this is the right typing, no? -- if mypy is complaining about it in other place, is better to fix in the right place